<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSet</class>
 <widget class="QDialog" name="GameSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>人人对战</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>人机对战</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>先手</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="playerFrist">
        <property name="text">
         <string>玩家</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="AiFrist">
        <property name="text">
         <string>AI</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>网络对战</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="serverRadio">
        <property name="text">
         <string>服务端</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="clientRadio">
        <property name="text">
         <string>客户端</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>311</width>
       <height>46</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>服务端</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>端口：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="serverPort">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>6666</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="serverListen">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>侦听</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>311</width>
       <height>46</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>客户端</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>IP地址：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="clientIP">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>端口：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="clientPort">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>6666</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="clientConnect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>连接</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="comfirm">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="pvpSelet">
      <property name="text">
       <string>人人对战</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="pveSelet">
      <property name="text">
       <string>人机对战</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="networkSelet">
      <property name="text">
       <string>网络对战</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
